src/pages/People.tsx

```diff
@@ -0,0 +1,90 @@
+import { useSearchParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+
+import { PeopleFilters } from '../components/PeopleFilters/PeopleFilters';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable/PeopleTable';
+
+import { filterPeople } from '../utils/filterHelper';
+import { getPeople } from '../api';
+
+import { Person } from '../types';
+import { sortPeople } from '../utils/sortHelper';
+
+export const People = () => {
+  const [searchParams] = useSearchParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(peopleFromServer => {
+        const peopleToDisplay = peopleFromServer
+          .map(person => ({
+            ...person,
+            mother: peopleFromServer.find(
+              somebody => somebody.name === person.motherName,
+            ),
+            father: peopleFromServer.find(
+              somebody => somebody.name === person.fatherName,
+            ),
+          }));
+
+        setPeople(peopleToDisplay);
+      })
+      .catch(() => {
+        setIsError(true);
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  const filteredPeople = filterPeople(people, searchParams);
+  const sortedPeople = sortPeople(filteredPeople, searchParams);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="columns is-desktop is-flex-direction-row-reverse">
+          {people.length > 0 && (
```

Comment:
This condition would be enough. The same in the code lower. Why don't you use it?
```suggestion
          {people.length && (
```

Reply:
@dellachaise Not, it not the same. If we use `!people.length` we will literally get **boolean type**. But in case of `people.length` we will get **number type**. That's why when our array will be empty we will have number 0 rendered on page. You can try and check it)

Reply:
Ok, I understand you 👍  I think you could leave it the way you have or try `!!`.